views/editprofile.view.php

```diff
@@ -4,72 +4,59 @@
-    <label for="confirm_password">Confirm Password</label>
-    <input type="password" name="confirm_password" id="confirm_password" value="<?= $user['password'] ?>">
-    <label for="isAdmin">Admin</label>
-    <input type="checkbox" name="isAdmin" id="isAdmin" value="<?= $user['isAdmin'] ?>">
-    <input type="submit" value="Submit">
-</form>
-<?php require $_SERVER['DOCUMENT_ROOT'] . '/views/partials/footer.php'; ?>
+<!-- input fields and edit buttons for all categories below -->
 
-
-    
-
-    <main>
-    <!-- input fields and edit buttons for all categories below -->
+<section>
     <h1> Edit Profile</h1>
-    
-    <section>
-    <h1>Personal Information</h1> 
-    <form action="/placeholder">
-    <label for="email">First name:</label>
-    <input type="text" placeholder="John"> <br>
-    <label for="wachtwoord">Last name:</label>
-    <input type="text" placeholder="Van den Berg"> <br>
-<form action="/placeholder">
-<label for="Voornaam">Email:</label>
-<input type="text" Placeholder="JohnvanDenBerg@gmail.com"> <br>
-<label for="Achternaam">Password:</label>
-<input type="text" placeholder="**********"> <br>
-<label for="Achternaam"> Confirm Password:</label>
-<input type="text" Placeholder="**********"> <br>
-<form action="/placeholder">
-<textarea name="" id="" cols="30" rows="2" Placeholder="Hobbies"></textarea> <br>
-<form action="/placeholder">
-<textarea name="" id="" cols="30" rows="2" Placeholder="Job Experiences"></textarea> <br>
-<textarea name="" id="" cols="30" rows="2" Placeholder="Educations"></textarea> <br>
-<input type="submit" value="Submit">
-</form>
+    <h2>Personal Information</h2>
+    <form action="" method="post" enctype="multipart/form-data">
+        <img src="/views/public/images/<?= $profileimg ?>" alt="">
+        <label for="fileToUpload" id="uploadImage">
+            <span>Select Image <span class="material-symbols-rounded">
+                    add_photo_alternate
+                </span></span>
+            <!-- hidden file input that gets replaced by the span -->
+            <input type="file" name="fileToUpload" id="fileToUpload" accept="image/*">
+        </label>
+        <input type="submit" value="Upload" name="uploadpfp">
+    </form>
+    <form action="" method="post">
+        <label for="FirstName">First name:</label>
+        <input type="text" placeholder="First Name" name="firstName" value="<?= updateUserInfo()["firstName"] ?>"> <br>
+        <label for="LastName">Last name:</label>
+        <input type="text" placeholder="Last Name" name="lastName" value="<?= updateUserInfo()["lastName"] ?>"> <br>
+        <label for="Email">Email:</label>
+        <input type="text" Placeholder="Email" name="email" value="<?= updateUserInfo()["email"] ?>"> <br>
+        <a href="/forgot">Change Password</a>
+
+        <button>Add hobby</button>
+        <br>
+        <button>Add Job Experience</button>
+        <br>
+        <button> <a href=" /about"> Add education</a></button>
```

Comment:
De spatie in " /about" is hier belangrijk om weg te halen

Reply:
hij werkt wel, en k weet ook niet waarom die daar staat, was nog van djimairo

Reply:
okay :)